mdb-realm/deployApps/action.yml

```diff
@@ -18,14 +18,16 @@ inputs:
     required: true
     description: The private API key.
   appsPath:
-    required: true
+    required: false
     description: The path to the apps to import into MongoDB Realm.
   differentiator:
     required: true
     description: Provide a differentiator for the build - e.g. "{{ matrix.os }}-{{ matrix.node-version }}"  or "integration tests"
 outputs:
   deployedApps:
     description: "A base64-encoded json object keyed on the app name containing the app ids of the deployed apps - e.g. { \"app1\": \"app1-abcde\"}"
+  clusterName:
+    description: The name of the created cluster
```

Comment:
Will this fail if the name is already taken?

Reply:
This is an output variable and the name of the cluster is generated and should be unique. But yes - the action will fail if it tries to create a cluster that already exists (e.g. if we re-run the action without cleaning up a previous deployment).